tests/Domain/data/process_info_valid.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types = 1);
+
+return [
+    'commonName' => 'foobar.dev',
+    'requiresAttention' => false,
+    'validations' => [['some' => 'array', 'with' => 'validations info' ]],
+    'notes' => [['some' => 'array', 'with' => 'notes']],
```

Comment:
I'd like to see more specific testing data if we ever decide to add models for these.

- "dcv" is the most used validation
- for notes we can make a test note. 
- 2020-09-09T09:18:57Z is the timestamp format

Reply:
Thijs added it